bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -148,6 +172,10 @@
 		<label>Color Loop</label>
 		<description>The effect channel allows putting the bulb in a color looping mode.</description>
 		<category>ColorLight</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Light</tag>
```

Comment:
Sending "ON" to a "Control-Light" semantic item, I would expect to turn on some light (to make the room lit) and not to start some fancy effect.

Reply:
Tags removed